ElunaTemplate.h

```diff
@@ -339,17 +277,29 @@ class ElunaTemplate
         lua_pop(E->L, 1);
     }
 
-    template<typename C>
+    template<typename C = void>
     static void SetMethods(Eluna* E, ElunaRegister<C>* methodTable)
```

Comment:
By changing the second argument to `ElunaRegister<C> const(&methodTable)[N]` (add size_t N to template args) you can drop all empty `{ nullptr, METHOD_REG_NONE }` that signify end of array

Reply:
I tried this with a range based loop, and I had some weird behavior where globals were suddenly registered as if they were object specific. I will have to look into it again tomorrow.

Reply:
Worked well with an iterator for loop instead